src/Native/CpuMath/IntrinsicsUtils.cs

```diff
@@ -0,0 +1,796 @@
+﻿using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.X86;
+using System;
+
+namespace Intrinsics
+{    public class IntrinsicsUtils
```

Comment:
Many of these methods look duplicated from `SseIntrinsics.cs`?

Reply:
Yes, nice catch.  Sorry shouldn't have incorporate this `src\Native\CpuMath` folder in this PR.  Sorry for any inconvenience caused, and thank you for your understanding.